_data/hosting_providers.json

```
@@ -2448,5 +2448,15 @@
     "plan": "",
     "reviewed": "2020.09.28",
     "note": "Let's Encrypt available on all hosting, VPS, and dedicated server plans."
-  }
+  },
+  {
+    "name": "celltek",
+    "link": "https://celltek-server.de/",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "",
+    "plan": "",
+    "reviewed": "2019.5.31",
```

This should be roughly the current date. It's meant to track the last time someone reviewed the status of the HTTPS support here.
```suggestion
    "reviewed": "2020.11.13",
```

---

Thanks!